### ISSUE TITLE
Missing hint for variables with `SECRET` prefix

### ISSUE THREAD

**Issue by johha** [MicrosoftDocs/azure-devops-docs#10688]
Hi,

I could not find any hint/note in the docs that variables with `SECRET` prefix are ignored completely during runtime.
I found this information on some MS support page (page is not available anymore):

> The variable named “SECRET_KEY” will be ignored is because the “secret” is a variable prefix reserved by the system. Any variable that begins with “secret” will not be available to your tasks and scripts.

A similar section should be added to the official docs to avoid unexpected behaviors for developers.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 11d3469c-83c2-c2f6-41d0-08f82c330bbc
* Version Independent ID: a373c48d-6395-3002-d2d8-bbc1fea2e48e
* Content: [Predefined variables - Azure Pipelines](https://docs.microsoft.com/en-us/azure/devops/pipelines/build/variables?view=azure-devops&tabs=yaml)
* Content Source: [docs/pipelines/build/variables.md](https://github.com/MicrosoftDocs/azure-devops-docs/blob/master/docs/pipelines/build/variables.md)
* Product: **devops**
* Technology: **devops-cicd-build**
* GitHub Login: @juliakm
* Microsoft Alias: **jukullam**

**Comment by juliakm** [comment]
Thanks @johha! I'm not able to reproduce this. Can you share an example?

**Comment by johha** [comment]
I created a variable named `SECRET_KEY` in an existing variable group. The variable contains sensitive information and thus is using type 'secret'. When consuming `SECRET_KEY` in a pipeline it is not available (other variables of the same group can be consumed). After renaming `SECRET_KEY` to `MY_SECRET_KEY` and adjusting the pipeline it can be consumed.  
It appears that variables which start with `SECRET` are not forwarded or dropped by Azure Pipelines. See also the quote above which I found on 'developercommunity.visualstudio.com/' (original page not available anymore).

I would be great if the documentation would contain a hint that variables starting with `SECRET` will be ignored/dropped.
